azure-pipelines.yml

```diff
@@ -0,0 +1,96 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+variables:
+  - name: buildConfiguration
+    value: 'release'
+  - name: buildPlatform
+    value: 'anycpu'
+  - group: 'Prod Management Resources'
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+
+pool:
+  name: 'DAS - Continuous Integration'
+  demands: Agent.OS -equals Windows_NT
+
+workspace:
+  clean: all
+steps:
+- task: UseGitVersion@5
+  inputs:
+    versionSpec: 5.x
+    useConfigFile: true
+    configFilePath: GitVersion.yml
+
+- task: DotNetCoreCLI@2
+  displayName: Restore
+  inputs:
+    command: restore
+    projects: 'src/**/*.csproj'
+
+- task: DotNetCoreCLI@2
+  displayName: Build
+  inputs:
+    projects: 'src/**/*.csproj'
+    arguments: '--configuration $(buildConfiguration) --no-restore'
+      
+- task: DotNetCoreCLI@2
+  displayName: Test
+  inputs:
+    command: test
+    projects: '**/*.UnitTests.csproj'
+    arguments: '--configuration $(buildConfiguration) --no-build'
+
+- template: azure-pipelines-templates/dependency-check.yml@das-platform-building-blocks
+
+- task: DotNetCoreCLI@2
+  displayName: 'Publish API'
+  inputs:
+    command: publish
+    publishWebProjects: false
+    projects: 'src/SFA.DAS.ContentApi.Api/SFA.DAS.ContentApi.Api.csproj'
+    arguments: '--configuration $(buildConfiguration) --output $(build.artifactstagingdirectory)/publish --no-restore --no-build'
+
+- task: CopyFiles@2
```

Comment:
Could this task be moved and combined with the below CopyFiles@2 task?

Reply:
Sorted (And removed duplicate publish Artifact)